src/lib/modules/Loan/actions.js

```diff
@@ -103,7 +104,8 @@ export const performLoanAction = (
         type: ACTION_HAS_ERROR,
         payload: error,
       });
-      dispatch(sendErrorNotification(error));
+      // TODO: Fix sendErrorNotification and use it here.
+      dispatch(addNotification('Error', error.message, 'error'));
```

Comment:
for the reviewers:
using `sendErrorNotification` here resulted on a button without any error notification in case of API errors on the response
IMO the usage was wrong here (`sendErrorNotification` redirects to error page instead of informing about the error, which might be confusing while performing a loan action) - so potentially the comment above could be remove, lets see if the rest of the team agrees

Reply:
I think this has to be understood before removing it. `sendErrorNotification` is used extensively, there must be something wrong here?
If @Pineirin cannot figure it out, I will have to investigate...